purchase_work_acceptance_late_fine/models/account_move.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2020 Ecosoft Co., Ltd. (http://ecosoft.co.th)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+    _description = "Journal Entries"
+
+    late_wa_id = fields.Many2one(
+        comodel_name="work.acceptance",
+        string="Late WA",
+        domain=lambda self: self._get_fine_amount(),
+    )
+
+    def _get_fine_amount(self):
+        wa = self.env["work.acceptance"].search([("late_fine", ">", 0)])
+        return [("id", "in", wa.ids)]
```

Comment:
```suggestion
    def _domain_late_wa(self):
        wa = self.env["work.acceptance"].search([("late_fine", ">", 0)])
        return [("id", "in", wa.ids)]
```
Use method name that starts with `_domain`. More on coding standard.

Also, I think the logic is not enough here. We should make sure that, if that WA is already used in any active document, it must not show.
For example, if WA is already used in invoice (state != cancel) it must not show. Otherwise, use will use this WA multiple times (we don't want that).

Reply:
If partner is already selected, show only WA of selected partner.
If partner is not selected, show all valid WAs.